hal/ia32/_interrupts.S

```diff
@@ -28,22 +27,6 @@
 #define EAX_OFFSET (16 + (FPU_CONTEXT_SIZE))
 
 
-.macro _INTERRUPTS_MULTILOCKSET reg
-	xorl \reg, \reg
-1:
-	xchgl _interrupts_multilock, \reg
-	orl \reg, \reg
-	jz 1b
-.endm
-
-
-.macro _INTERRUPTS_MULTILOCKCLEAR reg
-	xorl \reg, \reg
-	incl \reg
-	xchgl _interrupts_multilock, \reg
-.endm
-
-
```

Comment:
Because with this change it causes hard to avoid deadlocks and I think that `threads_common.spinlock` may be doing its job. 

But to be honest, I am still unsure of this change.